apps/server/src/modules/tool/common/service/common-tool-validation.service.ts

```diff
@@ -29,41 +36,59 @@ export class CommonToolValidationService {
 		return isValid;
 	}
 
-	public checkForDuplicateParameters(validatableTool: ValidatableTool): void {
-		const caseInsensitiveNames: string[] = validatableTool.parameters.map(({ name }: CustomParameterEntry) =>
-			name.toLowerCase()
+	public checkCustomParameterEntries(loadedExternalTool: ExternalTool, validatableTool: ValidatableTool): void {
+		this.checkForDuplicateParameters(validatableTool);
+
+		const parametersForScope: CustomParameter[] = (loadedExternalTool.parameters ?? []).filter(
+			(param: CustomParameter) =>
+				(validatableTool instanceof SchoolExternalTool && param.scope === CustomParameterScope.SCHOOL) ||
+				(validatableTool instanceof ContextExternalTool && param.scope === CustomParameterScope.CONTEXT)
 		);
 
+		this.checkForUnknownParameters(validatableTool, parametersForScope);
+
+		this.checkValidityOfParameters(validatableTool, parametersForScope);
+	}
+
+	private checkForDuplicateParameters(validatableTool: ValidatableTool): void {
+		const caseInsensitiveNames: string[] = validatableTool.parameters.map(({ name }: CustomParameterEntry) => name);
```

Comment:
The functionality that a parameter with name "something" and "Something" recognized as duplicate is lost